Dockerfile

```diff
@@ -2,9 +2,7 @@ FROM registry.access.redhat.com/ubi8/ruby-27
 
 USER root
 RUN rpm -Uvh https://download.postgresql.org/pub/repos/yum/reporpms/EL-7-x86_64/pgdg-redhat-repo-latest.noarch.rpm \
-  && dnf update -y \
-  && dnf remove -y postgresql* \
-  && dnf install --setopt=skip_missing_names_on_install=False,tsflags=nodocs -y shared-mime-info postgresql12-12.13 postgresql12-devel-12.13 postgresql12-libs-12.13 \
+  && dnf install --setopt=skip_missing_names_on_install=False,tsflags=nodocs --skip-broken -y shared-mime-info postgresql13 postgresql13-devel postgresql13-libs \
```

Comment:
`dnf update` was failing all the time with:

```
Error: 
 Problem: cannot install the best update candidate for package libpq-devel-13.5-1.el8.x86_64
  - nothing provides perl(IPC::Run) needed by postgresql15-devel-15.3-2PGDG.rhel8.x86_64
  - nothing provides perl(IPC::Run) needed by postgresql14-devel-14.8-2PGDG.rhel8.x86_64
  - nothing provides perl(IPC::Run) needed by postgresql13-devel-13.11-2PGDG.rhel8.x86_64
  - nothing provides perl(IPC::Run) needed by postgresql12-devel-12.15-2PGDG.rhel8.x86_64
  - nothing provides perl(IPC::Run) needed by postgresql11-devel-11.20-2PGDG.rhel8.x86_64
(try to add '--skip-broken' to skip uninstallable packages or '--nobest' to use not only best candidate packages)
Error: error building at STEP "RUN rpm -Uvh https://download.postgresql.org/pub/repos/yum/reporpms/EL-7-x86_64/pgdg-redhat-repo-latest.noarch.rpm   && dnf update -y   && dnf remove -y postgresql*   && dnf install --setopt=skip_missing_names_on_install=False,tsflags=nodocs -y shared-mime-info postgresql13 postgresql13-devel postgresql13-libs   && dnf clean all   && rm -rf /var/cache/yum": error while running runtime: exit status 1
```

I did not find a way to add the package providing `perl(IPC::Run)`

Also, installing postgresql13 was also failing with the same error without `--skip-broken`.
This doesn't seem to affect the tests. We probably need to check whether the image built by this Dockerfile actually works well. But if the `rails db:setup` step succeeds I don't know why it wouldn't.

Reply:
I think that we only need postgres libraries here, moreover the postgres libraries that come with RHEL, not external ones that are not used in production.